bug: add missing hooks for the claim module breaks the simulation

**Describe the bug**

Staking and gov hooks for the claim module are missing in the `app.go`:

```go
app.StakingKeeper = *stakingKeeper.SetHooks(
   stakingtypes.NewMultiStakingHooks(
      app.DistrKeeper.Hooks(),
      app.SlashingKeeper.Hooks(),
      app.ClaimKeeper.NewMissionDelegationHooks(missionIDStaking),
   ),
)

// register the gov hooks
app.GovKeeper = *app.GovKeeper.SetHooks(
   govtypes.NewMultiGovHooks(
      app.ClaimKeeper.NewMissionVoteHooks(missionIDVoting),
   ),
)
```

Now the simulation is properly running but starts to break:

```
panic: kv store with key <nil> has not been registered in stores
```

**To Reproduce**
Steps to reproduce the behavior:
1. Add the hooks in the `app.go`
2. Run the simulation: 
```shel
go test -benchmem -run=^$ -bench ^BenchmarkSimulation ./app -NumBlocks=200 -BlockSize 50 -Commit=true -Verbose=true -Enabled=true
```

--- a/app/app.go
+++ b/app/app.go
@@ -97,6 +97,11 @@ const (
 	AccountAddressPrefix = "cosmos"
 	Name                 = "testapp"
 	DefaultChainID       = "testapp-0"
+
+	// missionIDStaking is the mission ID for staking mission to claim airdrop
+	missionIDStaking = 1
+	// missionIDVoting is the mission ID for voting mission to claim airdrop
+	missionIDVoting = 2
 )
 
 // this line is used by starport scaffolding # stargate/wasm/app/enabledProposals
@@ -365,12 +370,6 @@ func New(
 		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
 	)
 
-	// register the staking hooks
-	// NOTE: stakingKeeper above is passed by reference, so that it will contain these hooks
-	app.StakingKeeper = *stakingKeeper.SetHooks(
-		stakingtypes.NewMultiStakingHooks(app.DistrKeeper.Hooks(), app.SlashingKeeper.Hooks()),
-	)
-
 	// Create evidence Keeper for to register the IBC light client misbehaviour evidence route
 	evidenceKeeper := evidencekeeper.NewKeeper(
 		appCodec,
@@ -419,6 +418,23 @@ func New(
 		app.BankKeeper,
 		app.DistrKeeper,
 		authtypes.FeeCollectorName,
+	)
+
+	// register the staking hooks
+	// NOTE: stakingKeeper above is passed by reference, so that it will contain these hooks
+	app.StakingKeeper = *stakingKeeper.SetHooks(
+		stakingtypes.NewMultiStakingHooks(
+			app.DistrKeeper.Hooks(),
+			app.SlashingKeeper.Hooks(),
+			app.ClaimKeeper.NewMissionDelegationHooks(missionIDStaking),
+		),
+	)
+
+	// register the gov hooks
+	app.GovKeeper = *app.GovKeeper.SetHooks(
+		govtypes.NewMultiGovHooks(
+			app.ClaimKeeper.NewMissionVoteHooks(missionIDVoting),
+		),
 	)
 
 	// this line is used by starport scaffolding # stargate/app/keeperDefinition

